### ISSUE TITLE
UI Bug in "Enable Replication" page

### ISSUE THREAD

**Issue by chrisvanmeer** [hashicorp/vault#16604]
**Describe the bug**
There appears to be a UI bug in the "Enable Replication" page on Vault Enterprise.  
In the Performance section of the screen, there is a big box surrounding the radio button, which looks off and prevents users from reading the explanation that is mentioned there.

**To Reproduce**
1. Log in to the UI
2. Go to _Status_
3. Click _Enable_ under the _Replication_ section
4. See the UI bug in the _Performance_ section 
  ![image](https://user-images.githubusercontent.com/3852620/183049281-824e28ee-5068-4c2b-ba2f-d7ead36c7f29.png)

**Expected behavior**
1. Log in to the UI
2. Go to _Status_
3. Click _Enable_ under the _Replication_ section
4. See consistency without the bug in the _Performance_ section 
  ![image](https://user-images.githubusercontent.com/3852620/183049609-25b7d00b-83c8-451c-8a44-dbf09deb0b4c.png)

**Environment:**
* Vault Server Version (retrieve with `vault status`):
  * `Version                  1.11.2+ent`
* Vault CLI Version (retrieve with `vault version`):
  * `Vault v1.11.2+ent (70297f44cf8a5f430a43b05fba32a4496e7b3b93), built 2022-07-29T19:03:34Z`
* Server Operating System/Architecture:
  * `Ubuntu 20.04.4 LTS (Focal Fossa) x86_64`

**Proposed fix**
Looking at the source code, a snippet from the _Performance_ section is

```html
<div>
  <div class="box columns is-centered">
    <input id="performance" name="replication-mode" type="radio">
    <label for="performance" data-test-replication-type-select="performance"></label>
  </div>
</div>
```

That second `div` needs to be removed because its classes are already inherited from the `label` that is surrounding this content (not in the snippet). Like so

```html
<div>
  <input id="performance" name="replication-mode" type="radio">
  <label for="performance" data-test-replication-type-select="performance"></label>
</div>
```

**Comment by chrisvanmeer** [comment]
Created first PR in here.

**Comment by chrisvanmeer** [comment]
It has been fixed in 1.11.3+ent.
